app/templates/search.hbs

```diff
@@ -1,2 +1,40 @@
-Search Page
-{{outlet}}
+
+<div id="bg">
+  <img src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/welcomeBG3.jpg"/>
+</div>
+<div class="wrapper">
+<img class="search-page-logo" src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/waldo_logoColor.png"/>
+
+<h3 class="search-page-slogan"> Let's find the job you've been looking for. </h3>
+
+<div class="div-search-bar">
+{{search-bar
```

Comment:
Maybe for the search bar and the up slider you could use the real images? I think the search logo is the white search logo and you can put it ontop of a red div